plugin/src/test/java/org/opensearch/ml/rest/MLCommonsRestTestCase.java

```diff
@@ -5,15 +5,8 @@
 
 package org.opensearch.ml.rest;
 
-import static org.opensearch.commons.ConfigConstants.OPENSEARCH_SECURITY_SSL_HTTP_ENABLED;
-import static org.opensearch.commons.ConfigConstants.OPENSEARCH_SECURITY_SSL_HTTP_KEYSTORE_FILEPATH;
-import static org.opensearch.commons.ConfigConstants.OPENSEARCH_SECURITY_SSL_HTTP_KEYSTORE_KEYPASSWORD;
-import static org.opensearch.commons.ConfigConstants.OPENSEARCH_SECURITY_SSL_HTTP_KEYSTORE_PASSWORD;
-import static org.opensearch.commons.ConfigConstants.OPENSEARCH_SECURITY_SSL_HTTP_PEMCERT_FILEPATH;
-import static org.opensearch.ml.common.MLTask.FUNCTION_NAME_FIELD;
-import static org.opensearch.ml.common.MLTask.MODEL_ID_FIELD;
-import static org.opensearch.ml.common.MLTask.STATE_FIELD;
-import static org.opensearch.ml.common.MLTask.TASK_ID_FIELD;
+import static org.opensearch.commons.ConfigConstants.*;
+import static org.opensearch.ml.common.MLTask.*;
```

Comment:
We should avoid import *.

Reply:
+1, but this only affects tests here so should be fine to get rid of it in the next PR.